### DISCUSSION TITLE
Cron Jobs for Django Tenants

### DISCUSSION THREAD

**Discussion by jairoshKumar** [django-tenants/django-tenants#907] [Q&A]
I have used django-tenants widely for my project. I have also used django-crontab in another projects. Have been trying to run cron jobs on my multi-tenant project but failing to do so. I am simply fetching the user data and printing the user.username in logs to check the value but no logs are created. I figured the reason jobs aren't running is maybe due to the absence of tenant/schema name while running the job, so it doesn't understand which tenant to fetch the user data from. If anyone has tried to work on a similar thing before and came to a solution please let me know, or if you've tried a certain way, I'll be happy to try it myself too.

Thank you. :)

**Comment by tomturner** [comment]
https://github.com/maciej-gol/tenant-schemas-celery
Not sure if this library will help you. I personally don't use use it with django-tenants however lots of people do
